### PR TITLE
approaching by adding conditions

### PR DESCRIPTION
**Pull request by KlausMikhaelson**
## Fixes Issue
It's a draft pr I want some help with the approach so I tried to approach it by adding conditionals but I am not sure why it's not working so can anyone help me out with it ?

## Check List (Check all the applicable boxes) 

- [x ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x ] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/good-first-issue-finder/pull/225"></a>

### CODE REVIEW

File: src/lib/components/issue-card.svelte [hunk 1]
```diff
@@ -37,6 +37,9 @@
         >
       </div>
     </div>
+    {#if `https://github.com/${issue.repository.owner.login}/${issue.repository.name}/blob/main/LICENSE`}
+      <span>True</span>
+    {/if}
```
Comment (Cahllagerfeld) [inline src/lib/components/issue-card.svelte:42]: I think this change doesn't work unfortunately, you are just checking a string, and not the actual repo.

What you would need to do, is extend the GraphQL-Query and retrieve the data from the Github-API

**Review by Cahllagerfeld — CHANGES_REQUESTED** [review]
Thanks for your contribution 🚀 
See my comment below :)

File: src/lib/components/issue-card.svelte (hunk 1 above)
Comment (Cahllagerfeld) [reply src/lib/components/issue-card.svelte:42]: For testing the GraphQL-Query, you can use the Explorer from Github [here](https://docs.github.com/en/graphql/overview/explorer)

Comment (KlausMikhaelson) [reply src/lib/components/issue-card.svelte:42]: yeah makes sense I just thought of trying this, thank you this will work !

**Pull request closed without merging**